logging/config/cluster-logging-log-forwarding.adoc

```diff
@@ -0,0 +1,38 @@
+:context: cluster-logging-log-forwarding
+[id="cluster-logging-log-forwarding"]
+= Forwarding cluster logs to specific endpoints
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+The cluster logging *Log Forwarding* feature enables administrators to configure custom pipelines to send your container and node logs to specific endpoints within or outside of your cluster. You can send logs by type to the internal {product-title} Elasticsearch instance and/or remote destinations not managed by {product-title} cluster logging, such as your existing logging service, an external Elasticsearch cluster, external log aggregation solutions like Splunk and Graylog, or a Security Information and Event Management (SIEM) system.
+
+Log Forwarding provides an easier way to forward logs to specific endpoints inside or outside your cluster than using the xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[Fluentd plugins].
+
+[NOTE]
+====
+The Log Forwarding feature is optional. If you do not want to forward logs and use only the internal {product-title} Elasticsearch instance, do not configure the Log Forwarding feature. 
```

Comment:
Maybe mention here also that it's tech preview.

Reply:
Outdated